### ISSUE TITLE
Low scores

### ISSUE THREAD

**Issue by mykolamelnykml** [ruifcruz/sroie-on-layoutlm#1]
I tried to run notebook on google colab and got:

f1 = 0.7409200968523002
loss = 0.12807334628370073
precision = 0.7814096016343207
recall = 0.7044198895027625

Could you please help to reproduce your results?

**Comment by ruifcruz** [comment]
Hi! Have you tried changing the the variable num_attention_heads to 8 (on the config.json of the model)?

**Comment by mykolamelnykml** [comment]
Hello,
Yes I tried num_attention_heads=8. I used model from Microsoft site. So looks like it is different.

I reproduced your results using model from the Kaggle site.

Thank you.

**Comment by ruifcruz** [comment]
Ok, it is a bit strange, I wasn't aware of this difference. Thanks for your
feedback.

**Comment by ninjakx** [comment]
@kolia1985 : Is the notebook working for you? I am getting some erros related to preprocessing of SROIE data.

`    sent, labels, bbox = define_labels(pos, sent, labels, bbox, class_value, classification)  
`

In this line.
